data/human/sagassaga.txt

```diff
@@ -0,0 +1,3060 @@
+			choice
+				`	(Investigate the Noise.)`
+					goto investigate
+				`	(Ignore the Noise.)` 
+			`The doings of sheep, no matter how odd or murderous their bleats may sound, are not exactly a pressing matter for you. You have more important things to do, and make your way through the throng of haggling hucksters, and back toward your ship.`
+				decline
+			label investigate
+			`	You push through the exit, and out into the rolling sheep filled fields of New Kansas. While there are pockets of trees and shrubbery, for much of the view you see nothing, but an endless expanse of grazing land covered in the aimless dots of sheep.`
+			`	The noise that brought you out into this ebbing dusk is louder now, and quite near. Just beyond a few hedges you see a couple of red-eyed homicidal sheep, with oddly sharp fangs bared, growling and stamping at a huddled animal in a hole.`
+			choice
+				`	(Run and yell at the sheep.)`
+					goto yelling
+				`	(Leave.)`
+			`	You've heard about the murderous sheep of New Kansas. You have no desire to end your career as another victim of their killing spree. Discretion, being the better part of valor, you quietly return to the safety of the spaceport.`
+				decline
+			label yelling
+			`	For one terrifying instant the mad sheep hold their ground, and bare their oddly sharp fangs at you, but they hesitate, and then turn and flee.`
+			`	With the assailants dealt with, you turn your gaze to the hole, and the sheep's victim. You find a dog, weak, yet thrilled to see you.`
+			scene "scene/doghole"
+			`	The dog crawls out of its hole, and licks you eagerly. You notice a worn, and threadbare collar with the barely legible name, 'Saga' scrawled on it.`
+			choice
+				`	(Try to find Saga's owner.)`
+					goto owner
+				`	(Shoo the dog away.)`
+			`	Saga clearly wants to follow you, her savior, wherever you're headed. But after yelling at her a few times she finally drops are tail and slinks away into the shrubbery. Last thing you need is some lost mutt following you around. You ignore the sound of her low whimpers as you return to the spaceport.`
+				decline
+			label owner
+			`	Saga happily follows you inside the spaceport. No one seems to mind a mutt at your heels, and after a little searching you find an information kiosk. A man with tired eyes, and a wool vest stands within it.`
+			choice
+				`	(Inquire about the dog, and her possible owner.)`
+			`	The man's eyes seem to sag as you speak. Your earnest words lulling him to sleep. Once you're done he frowns, and shakes his head slowly.`
+			`	"Ser," he says with forced politeness, "this is a spaceport, not a shelter. We are aware of that stray dog. She has been prowling around for weeks. No one has come to collect her, and no one wants her. Frankly, we assumed the feral sheep would get to her by now. It would have done us all a favor.`
```

Comment:
To make the spellcheck stop complaining I think you need to add the whole line (with indents) to the `codespell.exclue` file.